proto/panacea/market/v2/tx.proto

```diff
@@ -27,3 +29,29 @@ message MsgCreateDeal {
 message MsgCreateDealResponse {
   uint64 deal_id = 1;
 }
+
+//MsgSellData defines the Msg/SellData request type.
+message MsgSellData {
+  DataValidationCertificate cert = 1;
+  string seller = 2;
+}
+
+//MsgSellDataResponse defines the Msg/SellData response type.
+message MsgSellDataResponse {
+  cosmos.base.v1beta1.Coin reward = 1;
+}
+
+// DataValidationCertificate defines data validation certificate.
+message DataValidationCertificate {
+  UnsignedDataValidationCertificate unsigned_cert = 1;
+  bytes signature = 2;
+}
+
+// UnsignedDataValidationCertificate defines unsigned data validation certificate.
+message UnsignedDataValidationCertificate {
+  uint64 deal_id = 1;
+  string data_hash = 2;
+  string encrypted_data_url = 3;
+  string data_validator_address = 4;
+  string requester_address = 5;
+}
```

Comment:
I have a question. If ```signature``` is Base64 encoded, is the data type of ```signature``` will be ```string```?

Reply:
Yes, that's right. Base64 encoding increases the length, but replaces the byte array with a human-readable string.
You can take a look at an example [here](https://github.com/medibloc/panacea-data-market-validator/pull/21/files#diff-c70ba4f8c966bc19cae21720247c4a3e51f8b1d1fa6fae21136dbc981ced8debR11).

note: https://en.wikipedia.org/wiki/Base64

Reply:
In my opinion, I think that change ```[]byte``` to ```string``` would be better!

Reply:
It seems that type of signature is defined in bytes in other `.proto` file of cosmos-sdk.
For consistence and ease to verify signature, I think bytes would be better.
In case of `data_hash` and `encrypted_data_url`, I think we can use either `string` or `bytes` for our convenience.

Reply:
I will apply the conclusion of this discussion in next PR after resolve other reviews.

Reply:
@inchori @hansol-medi 
Thank you for your comments. What I am concerned about is that byte arrays are not easy to express in json format.
When the seller has refined data in json format, if the signature is in the byte array format, it will probably have a long array of numbers.
```json
{
    "signature":[10, 9, 11, 20, 18]
}
```
If we base64 encode it, it will show up as a human-readable string
```json
{
    "signature": "asdfeewqcv..."
}
```

In general, in json, I don't use a byte array, but I think that a base64 encoded string isused.

Reply:
Ah, I'm thinking again, proto is not a place to specify json. 😅
However, the client needs to think differently.(Where the json file is handled directly)

Reply:
So if the ```signature``` field is Base64 encoded, the data type of ```signature``` will be changed ```parse.go``` file .